app/controllers/country_controller.rb

```diff
@@ -8,9 +8,14 @@ def show
     @country_presenter = CountryPresenter.new @country
 
     @flag_path = ActionController::Base.helpers.image_url("flags/#{@country.name.downcase}.svg"),
-                 @iucn_categories = @country.protected_areas_per_iucn_category
+    @iucn_categories = @country.protected_areas_per_iucn_category
     @governance_types = @country.protected_areas_per_governance
 
+    designations = @country_presenter.designations
```

Comment:
I feel this should be an instance variable instead that a local one.
When using the `stats-designations` partial in the country show view we could then pass `@designations` instead of calling `@country_presenter.designations` again, saving some time and computation